### PR TITLE
modernize webpack | refactor

### PR DESCRIPTION
**Pull request by jbetancur**

### CODE REVIEW

File: templates/default/karma.conf.js [hunk 1]
```diff
@@ -1,56 +1,61 @@
...
             'node_modules/babel-polyfill/dist/polyfill.min.js',
             'node_modules/sinon/pkg/sinon.js',
             'node_modules/jasmine-sinon/lib/jasmine-sinon.js',
-            'node_modules/jasmine-expect/dist/jasmine-matchers.js',
             'node_modules/angular/angular.js',
             'node_modules/angular-mocks/angular-mocks.js',
-        <% if (materialize) { %>
-        'node_modules/angular-material/angular-material-mocks.js',
-        <% } %>
+            'node_modules/angular-material/angular-material.js',
```
Comment (digitalbs) [inline templates/default/karma.conf.js:54]: We do not want to load these if we are not using the material library

File: templates/default/karma.conf.js [hunk 2]
```diff
@@ -62,25 +67,25 @@ module.exports = function(config) {
...
-                preLoaders: [{
-                    test: /\.js$/,
-                    include: srcPath,
-                    exclude: /(test)/,
-                    loader: 'isparta-loader'
-                }]
+                loaders: webpackSettings.getLoaders(webpackConfig, ExtractTextPlugin)
+            },
+            sassLoader: {
+                includePaths: [pathToBourbon]
```
Comment (digitalbs) [inline templates/default/karma.conf.js]: Are we including Bourbon?

Comment (jbetancur) [reply templates/default/karma.conf.js]: nice catch : removing bourbon

File: templates/default/karma.conf.js (hunk 1 above)
Comment (jbetancur) [reply templates/default/karma.conf.js:54]: ill fix

**Pull request merged**